src/v/cluster/tx_gateway.cc

```diff
@@ -41,7 +42,10 @@ tx_gateway::try_abort(try_abort_request&& request, rpc::streaming_context&) {
 ss::future<init_tm_tx_reply>
 tx_gateway::init_tm_tx(init_tm_tx_request&& request, rpc::streaming_context&) {
     return _tx_gateway_frontend.local().init_tm_tx_locally(
```

Comment:
it looks like we never dispatch init_tm_tx RPC (if we do then why don't we need to pass expected_pid?), please create an issue to double check that we don't dispatch init_tm_tx and to remove that RPC and add a comment to that issue here

Reply:
No
We did it